src/main/java/org/jenkinsci/plugins/ssegateway/Endpoint.java

```diff
@@ -39,6 +39,8 @@
 import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.StaplerResponse;
 import org.kohsuke.stapler.interceptor.RequirePOST;
+import org.slf4j.Logger;
```

Comment:
I thought Jenkins used `java.util.logging` instead of `slf4j`? Do we have an slf4j-to-java.util.logging converter in core or something?

Reply:
tested with 2.176.2 and it works :) 
core has ` slf4j-jdk14 ` and it's used for long time now because of maven(-plugin) dependencies.